chimneyCats/src/test/scala/io/scalaland/chimney/cats/CatsValidatedSpec.scala

```diff
@@ -16,7 +16,7 @@ object CatsValidatedSpec extends TestSuite {
 
   val tests = Tests {
 
-    "transform always succeeds" - {
+    test("transform always succeeds") - {
```

Comment:
Any reason for using different syntax than the rest of the library?

Reply:
Ah sorry, I should have converted the rest of them as well, I was getting deprecated syntax for the existing tests so I wanted to move to the new syntax. Hope that’s alright

Reply:
No worries, the best would be to switch to the new syntax in a separate PR.

Reply:
Sounds good! I'll switch to the new syntax in a separate PR